src/utils/testing/mocks/mockAssignedTask.ts

```diff
@@ -1,4 +1,4 @@
-import { mockObject } from '.';
+import { mockObject } from './index';
```

Comment:
Aren't these synonymous? Why do you think this is better?

Reply:
This will have been done automatically by my IDE - I will check for other occurrences :-D